src/framework/components/anim/component.js

```diff
@@ -369,6 +369,9 @@ class AnimComponent extends Component {
         if (typeof value === 'string') {
             const entity = this.entity.root.findByGuid(value);
             if (entity) this.data.rootBone = entity;

```

Comment:
I'd probably change this to simply
`this.data.rootBone = entity`
so that it gets set to null if not found .. otherwise the previous value will stay assigned? and do 
`if (!entity) console.warning ...`